blue_hardware_interface/src/blue_hardware_interface.cpp

```diff
@@ -46,10 +46,7 @@ BlueHW::BlueHW(ros::NodeHandle &nh) : nh_(nh) {
   for (auto id : params_.motor_ids)
     motor_commands_[id] = 0.0;
 
-  // Load in some initial values
-  read();
-
-  // Calibration services
+  // Calibration service
```

Comment:
@gbalke What was the reason for removing `read()` again?

Reply:
+1

Reply:
The read is never actually used.

Reply:
If you don't call read() before advertising the joint calibration service, there's a chance that the calibration service (eg joint snapping) will be called under the assumption that all the joint values are zero/unpopulated

Reply:
Fixed 72c053b